server.go

```diff
@@ -586,44 +587,76 @@ func (s *Server) ReadMetricSocket(serverConn net.PacketConn, packetPool *sync.Po
 }
 
 // ReadTraceSocket listens for available packets to handle.
-func (s *Server) ReadTraceSocket(packetPool *sync.Pool) {
+func (s *Server) ReadTraceSocket(serverConn net.PacketConn, packetPool *sync.Pool) {
```

Comment:
Nit: While you're here, change this to `ReadSpanSocket` since that's what it's really doing?

Reply:
Renamed to `ReadSSFPacketSocket` (: